tools/ingest-converter/src/main/java/org/logstash/ingest/JsUtil.java

```diff
@@ -12,7 +12,7 @@
     /**
      * Script names used by the converter in correct load order.
      */
-    private static final String[] SCRIPTS = {"shared", "date", "grok", "geoip", "pipeline"};
+    private static final String[] SCRIPTS = {"shared", "date", "grok", "geoip", "pipeline", "convert"};
```

Comment:
@suyograo hah I was wrong (obviously ... in hindsight) ... the load order doesn't matter :) `"pipeline"` uses `"convert"` logic just fine :)